src/main/java/training/subsystems/ArmSubsystems/elbowSubsystem/simulationElbow/SimulationElbow.java

```diff
@@ -0,0 +1,95 @@
+package training.subsystems.ArmSubsystems.elbowSubsystem.simulationElbow;
+
+import edu.wpi.first.math.controller.ArmFeedforward;
+import edu.wpi.first.math.geometry.Rotation2d;
+import training.RobotConstants;
+import training.subsystems.ArmSubsystems.elbowSubsystem.ElbowInputsAutoLogged;
+import training.subsystems.ArmSubsystems.elbowSubsystem.IElbow;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.SingleJointedArmSim;
+import training.subsystems.ArmSubsystems.elbowSubsystem.ElbowConstants;
+
+public class SimulationElbow implements IElbow {
+
+    private SingleJointedArmSim motor;
```

Comment:
remove instance, but generally the order of vars is:
static vars
final vars
vars